Yoast/yoast-components
# Added explanatory text for the replacevars

## Summary

This PR can be summarized in the following changelog entry:

* Added explanatory text for the replacevars in the snippet editor.

## Relevant technical choices:

* Added the props for the explanation, so that it can be overwritten should the user not want add it. Set the explanation we want to use as the default prop.
* No styles yet. So chosen style is based on surroundings:

<img width="663" alt="add_new_post_ _local_wordpress_test_ _wordpress-2" src="https://user-images.githubusercontent.com/11849359/40476726-08feea48-5f45-11e8-9696-638d0fc17046.png">


## Test instructions

This PR can be tested by following these steps:

* Visit the post page.
* Check if the help text provided in the issue is added between the meta description field and the close editor button.
* Check if tests pass and if it still looks good on scaled screens.

Fixes https://github.com/Yoast/wordpress-seo/issues/9786
